doc/tutorials/klondike/app/lib/step5/components/card.dart

```diff
@@ -0,0 +1,446 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/events.dart';
+import 'package:flutter/animation.dart';
+
+import '../klondike_game.dart';
+import '../pile.dart';
+import '../rank.dart';
+import '../suit.dart';
+import 'stock_pile.dart';
+import 'tableau_pile.dart';
+
+class Card extends PositionComponent with DragCallbacks, TapCallbacks {
```

Comment:
You can use `HasWorldReference` here instead of having to pass it in:
```suggestion
class Card extends PositionComponent with DragCallbacks, TapCallbacks, HasWorldReference<KlondikeWorld> {
```